eos-knowledge.gresource.xml

```diff
@@ -76,6 +76,7 @@
     <file>js/app/dispatcher.js</file>
     <file>js/app/dominantColor.js</file>
     <file>js/app/downloader.js</file>
+    <file>js/app/entryPoints.js</file>
```

Comment:
Seems like the entryPoints.js file itself should be part of this commit, rather than the previous one.